maestro-android/build.gradle

```diff
@@ -30,7 +30,7 @@ android {
     compileSdk 31
 
     defaultConfig {
-        applicationId "dev.mobile.conductor"
+        applicationId "dev.mobile.maestro"
```

Comment:
Are we planning to do a scratch release with 1.0.0?

Reply:
Yeah, figured might as well reset the counter